utils/getTzktAlias.ts

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
```

Comment:
Can you move this hook to the file `tzktHooks.ts` ? So that everything related to tzkt is in the same file

I never in typescript if we should use `<string| undefined>` or just `string?`